packages/frontend-api/src/Resources/config/graphql-types/ModelType/Slider/SliderItemDecorator.types.yaml

```diff
@@ -0,0 +1,36 @@
+SliderItemDecorator:
```

Comment:
I would not inlcude these changes into the original commit (now it is a fixup), rather create a separate commit with message something like "FE API: move (part of) Slider Item type definition from project-base to the frontend-api package"

This can be done during the interactive rebase, so let me know if you need help with that :wink: